api/src/services/db-doc.js

```diff
@@ -45,35 +56,42 @@ module.exports = {
   // this filters audited endpoints, so valid requests are allowed to pass through to the next route
   filterOfflineRequest: (userCtx, params, method, query, body) => {
     const isAttachment = params.attachmentId;
+    let stored;
+    let requested;
```

Comment:
Nit: as below - replace `let` with `const`. If it becomes too nested then feel free to pull out a function which takes these docs as params.